### ISSUE TITLE
Error running main_ADMM_CSNet_test.m

### ISSUE THREAD

**Issue by olgudel** [yangyan92/ADMM-CSNet#2]
When running each line of main_ADMM_CSNet_test.m in CSNet-Image, I get an error on the following line:

 **[re_LOss, re_PSnr , re_SSim, rec_image ]= loss_with_gradient_single_before(x0,  im_ori,  net, mask.picks, mask.perm);**

> Attempt to execute SCRIPT vl_nnconv as a function:
> C:\Users\Oliver\Desktop\ADMM-CSNet-master\matlab\vl_nnconv.m
> Error in dagnn.Conv2D/forward (line 10)
             outputs{1} = vl_nnconv(inputs{1}, params{1}, params{2}, 'pad', pad1);
> Error in dagnn.Layer/forwardAdvanced (line 85)
       outputs = obj.forward(inputs, {net.params(par).value}) ;
> Error in dagnn.DagNN/eval (line 91)
   obj.layers(l).block.forwardAdvanced(obj.layers(l)) ;
> Error in loss_with_gradient_single_before (line 41)
 net.eval(inputs) ; 

Is there a particular way to resolve this to test the code?  I'm running this on MATLAB R2018a.
